net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/00.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/01.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/02.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/03.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/04.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/05.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/06.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/julianwi/newpipe_flatpak/-/raw/623c65625d355138af677492b7496d9e128ed0c7/screenshots/newpipe/07.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>AudioVideo</category>
+  </categories>
+  <launchable type="desktop-id">net.newpipe.NewPipe.desktop</launchable>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">moderate</content_attribute>
+    <content_attribute id="drugs-alcohol">moderate</content_attribute>
+    <content_attribute id="drugs-narcotics">moderate</content_attribute>
+    <content_attribute id="drugs-tobacco">moderate</content_attribute>
+    <content_attribute id="language-profanity">moderate</content_attribute>
+    <content_attribute id="language-humor">moderate</content_attribute>
+    <content_attribute id="language-discrimination">moderate</content_attribute>
+    <content_attribute id="social-chat">moderate</content_attribute>
+    <content_attribute id="social-info">mild</content_attribute>
+    <content_attribute id="money-advertising">moderate</content_attribute>
+  </content_rating>
+  <releases>
+    <release version="0.27.2" date="2024-07-25">
+      <url type="details">https://example.org/changelog.html#version_1.0.1</url>
+      <description>
+        <p>Initial Flatpak version. Including all features from Android version</p>
+      </description>
```

Comment:
Add some actual description.

Reply:
I have replaced it with the official release description now